packages/react/src/use-rx.ts

```diff
@@ -54,13 +55,11 @@ export function useRx<T>(observable: WrappedObservable<T>, deps: any[] = [observ
 						setCount(c => c + 1)
 					}
 				}
-				// eslint-disable-next-line react-hooks/exhaustive-deps
 			}),
-		deps
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+		[...deps, memoized]
```

Comment:
Are you sure it's needed? It was intentionally not added there. It can break things I think

Reply:
reverted